api/src/main/java/jakarta/enterprise/concurrent/Schedule.java

```diff
@@ -25,18 +25,90 @@
 import java.time.Month;
 
 /**
- * <p>Defines schedules for
- * {@link Asynchronous#runAt() scheduled asynchronous methods}.</p>
+ * <p>Defines a schedule that indicates when to run a method.</p>
  *
  * <p>For the scheduled asychronous method to aim to run at a given day and time,
  * all of the criteria specified by the {@code Schedule} must match
  * or be disregarded according to the rules of each field.</p>
  *
+ * <h2>Scheduled methods</h2>
+ *
+ * <p>When the {@code Schedule} annotation is applied to a CDI managed bean method,
+ * the schedule defines the times after which to run the method. The method must
+ * have a {@code void} return type and no parameters. The bean must not be a
+ * Jakarta Enterprise Bean.</p>
+ *
+ * <p>Upon starting the application, the Jakarta EE Product Provider computes the
+ * next time from the {@code Schedule} annotation and schedules a task that aims to
+ * run at the computed time on the default {@link ManagedScheduledExecutorService}.
+ * After it is time to run the task, the task first checks to ensure that the
+ * schedule does not require {@linkplain #skipIfLateBy() skipping} the invocation
+ * of the method. If method invocation is not skipped, the task obtains an instance
+ * of the bean and invokes the method on the bean instance.</p>
+ *
+ * <p>After successful or skipped invocation of the method, the task uses the
+ * {@code Schedule} annotation to computes the next time that is after the method
+ * completion time (or the time of the skip) and likewise schedules another task
+ * that aims to run after the computed next time. This continues with each
```

Comment:
```suggestion
 * <p>After successful or skipped invocation of the method, the task uses the
 * {@code Schedule} annotation to compute the next time.
 * The computed next time is after the method completion time or the time of the skip.
 * The task schedules another task that aims to run after the computed next time.
 * This continues with each
```

Avoid a run-on sentence with ambiguous subjects.

Reply:
Thanks for the grammar improvements!